src/pybind/mgr/volumes/fs/operations/versions/subvolume_base.py

```diff
@@ -100,7 +100,7 @@ def load_config(self):
         else:
             self.metadata_mgr = MetadataManager(self.fs, self.config_path, 0o640)
 
-    def set_attrs(self, path, size, isolate_namespace, pool, uid, gid):
+    def set_attrs(self, path, size, isolate_namespace, pool, uid, gid, namespace=None):
```

Comment:
Is there a need to pass in the additional parameter? Can't that be inferred by passing a non-None namespace?